Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```diff
@@ -40,7 +40,11 @@ configuration:
 - defaultvalue: '-59'
   display: File Threshold
   name: file_threshold
+  type: 0
   required: false
+- display: Deletion Batch Size
```

Comment:
If it's only for one command why is it a param and not an argument of that function?

Reply:
Because it was supposed to be used in other commands as well, but it seemed to me after reviewing that this is the only command that uses the mechanism.
Obviously mentioned it in the issue - but no adequate response (they are very unresponsive and uninterested in this fix)

Reply:
i think that this should be done in an argument, seems un-natural this way